### PR TITLE
💄 Refactor sidebar toggle

### PR DESCRIPTION
**Pull request by timokohlmann**
Refactors the sidebar toggle

### CODE REVIEW

File: frontend/scss/components/atoms/sidebar-toggle.scss [hunk 1]
```diff
@@ -181,15 +143,32 @@ $originalEasing: cubic-bezier(0,0,.21,1);
...
+              transform: rotate(90deg) translateY(-110px);
               svg {
                 transform: rotate(180deg);
               }
             }
+
+            .label-title {
+              display: inline-flex;
+              padding-left: 10px;
+
```
Comment (robinvanopstal) [inline frontend/scss/components/atoms/sidebar-toggle.scss:162]: Remove empty line :)

File: frontend/scss/components/atoms/sidebar-toggle.scss [hunk 2]
```diff
@@ -181,15 +143,32 @@ $originalEasing: cubic-bezier(0,0,.21,1);
...
-            margin-left: -88px;
+            transform: rotate(-90deg);
+            top: 186px;
+            margin-left: -125px;
+
             .label-icon {
+              width: 35px;
+              margin-left: -2px;
+              transform: rotate(90deg) translateY(-110px);
               svg {
```
Comment (robinvanopstal) [inline frontend/scss/components/atoms/sidebar-toggle.scss:154]: Add an empty line before starting with new selectors 🥳

File: frontend/templates/views/partials/toc.j2 [hunk 3]
```diff
@@ -12,8 +12,8 @@
       <span class="ap-o-toc-hl">{{ _('Table of contents') }}</span>
       {# The icons direction depends on the inputs checked state #}
       <span class="nav-icon">
-        {% do doc.icons.useIcon('icons/angle-down-solid.svg') %}
-        <svg class="ap-a-ico"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#angle-down-solid"></use></svg>
+        {% do doc.icons.useIcon('icons/sidebar-toggle.svg') %}
+        <span class="label-icon"><svg><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#sidebar-toggle"></use></svg></span>
```
Comment (robinvanopstal) [inline frontend/templates/views/partials/toc.j2:16]: What do you need the extra `<span>` for?

**Comment by robinvanopstal** [comment]
One more thing I

 noticed:

**Comment by matthiasrohmer** [comment]
This will be superseded by Robin's frontend21 efforts.

**Pull request closed without merging**